packages/datagrid/src/basicmousehandler.ts

```diff
@@ -606,15 +606,8 @@ class BasicMouseHandler implements DataGrid.IMouseHandler {
     return Private.cursorMap[handle];
   }
 
-  /**
-   * Get the current pressData
-   */
-  get pressData(): PressData.PressData | null {
-    return this._pressData;
-  }
-
```

Comment:
Doesn't this mean that this property will not be available on the public API anymore ?

Reply:
Yeah agreed. Can this be added back?

Reply:
Anyone able to make a PR?

Reply:
I'll submit one

Reply:
@blink1073 to revert this, I'll need to restore the variable name of `pressData` back to `_pressData` as the `get pressData()` and `pressData` attribute have duplicate identifiers. Unless you have other suggestions?

Reply:
Done in #167